docs/svsm.md

```diff
@@ -0,0 +1,135 @@
+# SVSM  Utility Script (Launch_SVSM.sh)
+
+This 'Launch_SVSM.sh'  utility script is a provisioning and test script that performs three tasks:
```

Comment:
```suggestion
'Launch_SVSM.sh' is a provisioning and testing script that performs three tasks:
```

Reply:
Agreed. Thanks for pointing that out; I'll update it.